src/pybind/mgr/dashboard/module.py

```diff
@@ -779,7 +813,7 @@ def _get_mds_names(self, filesystem_id=None):
 
             @cherrypy.expose
             @cherrypy.tools.json_out()
-            def mds_counters(self, fs_id):
+            def mds_counters(self, mds_id):
```

Comment:
This really is a filesystem ID rather than an MDS ID (see below where it is getting the list of MDSs in the filesystem)